roadservice/core/models.py

```diff
@@ -233,13 +293,93 @@ def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
         self.type = Role.Type.COUNTY_MODERATOR
 
-    # TODO
-    def assign_county_expert(self):
-        pass
+    @property
+    def county(self):
+        return self.region.get_concrete()
 
     def get_concrete(self):
         return self
 
+    def add_service_team(self, speciality, team_members_users):
+        for team_member in team_members_users:
+            if team_member.has_role():
+                return None
+        new_team = ServiceTeam.objects.create(county=self.county, speciality=speciality)
+        for team_member in team_members_users:
+            Serviceman.objects.create(team=new_team, user=team_member)
+        return new_team
+
+    def edit_service_team(self, team, speciality, team_members_users):
+        ex_members_users = []
+        for member in team.members.all():
+            ex_members_users.append(member.user)
+        for user in team_members_users:
+            if user.has_role() and user not in ex_members_users:
+                return False
+        for user in team_members_users:
+            if user.has_role() and user in ex_members_users:
+                continue
+            Serviceman.objects.create(user=user, team=team)
+        for member in team.members.all():
+            if member.user not in team_members_users:
+                member.delete()
+        team.speciality = speciality
+        return True
+
+    def delete_service_team(self, team):
+        for member in team.members.all():
+            member.delete()
+        team.delete()
+
+    def add_speciality(self, name):
+        if not Speciality.objects.filter(name=name).exists():
+            Speciality.objects.create(name=name)
+        return Speciality.objects.get(name=name)
+
+    def rename_speciality(self, speciality, new_name):
+        speciality.name = new_name
+        speciality.save()
```

Comment:
You see, I'm not happy with the state of our return values. We have exceptions, we have boolean values (which show whether the operation was successful), and we sometimes return a particular object. This is not a problem by itself, but we don't have a standard. Would you agree that "renaming a speciality" looks very much like "editing a service team?" However, we have different types of return values for these two functions.

Reply:
I can see a justification for this. But let's make an explicit decision about our return values.

Reply:
I raised the same issue in our last meeting! We need a standard for this.